recipes/vispy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: vispy
+  version: 0.5.0
+
+source:
+  git_url: https://github.com/vispy/vispy.git
```

Comment:
Do you want latest master? I would package the stable released version first and then a "rc."